packages/bun/src/integrations/bunserver.ts

```diff
@@ -0,0 +1,143 @@
+import { captureException, getCurrentHub, runWithAsyncContext, startSpan, Transaction } from '@sentry/core';
+import type { Integration } from '@sentry/types';
+import {
+  addExceptionMechanism,
+  getSanitizedUrlString,
+  objectify,
+  parseUrl,
+  tracingContextFromHeaders,
+} from '@sentry/utils';
+
+function sendErrorToSentry(e: unknown): unknown {
+  // In case we have a primitive, wrap it in the equivalent wrapper class (string -> String, etc.) so that we can
+  // store a seen flag on it.
+  const objectifiedErr = objectify(e);
+
+  captureException(objectifiedErr, scope => {
+    scope.addEventProcessor(event => {
+      addExceptionMechanism(event, {
+        type: 'bun',
+        handled: false,
+        data: {
+          function: 'serve',
```

Comment:
Wasn't sure if `serve` or `fetch` is more correct but w/e

Reply:
yeah debated this too, will stick with serve for now, but we can always adjust later.